files/master.cfg

```diff
@@ -502,6 +505,18 @@ add_parked_builder('fedora29-x86_64', env=env_fedora, factory=Make(checking=True
 add_parked_builder('fedora30-x86_64', env=env_fedora, factory=Make(checking=True))
 add_parked_builder('fedora31-x86_64', env=env_fedora, factory=Make(checking=True))
 
+{% else %}
+{%     if afsbotcfg_testbuilders is not defined %}
+add_gerrit_dev('centos73-x86_64')
```

Comment:
I think we should move the data to the inventory and use jinja2 to create the the configuration.

Originally, we use python as a macro language to set the BuildmasterConfig dictionary.  But now we want to have different BuildmasterConfig dictionaries depending on the inventory host variables.

It might be better to split this large pull request into several so can migrate.